ops/phase1/run.sh

```diff
@@ -25,12 +25,12 @@ echo "Now copying this state to the remote"
 terraform init -force-copy
 terraform refresh -var "namespace=$1"
 
-export REGISTRY_NAME=$(terraform output operations_container_registry_login_server)
-export TF_VAR_resource_group_name=$(terraform output operations_resource_group_name)
-export TF_VAR_storage_account_name=$(terraform output operations_storage_account_name)
-export SUBNET_ID=$(terraform output operations_deployment_subnet_id)
-export OPERATIONS_VIRTUAL_NETWORK=$(terraform output operations_virtual_network)
-export LOGGING_WORKSPACE=$(terraform output logging_workspace_name)
+export REGISTRY_NAME=$(terraform output -raw operations_container_registry_login_server)
+export TF_VAR_resource_group_name=$(terraform output -raw operations_resource_group_name)
+export TF_VAR_storage_account_name=$(terraform output -raw operations_storage_account_name)
+export SUBNET_ID=$(terraform output -raw operations_deployment_subnet_id)
+export OPERATIONS_VIRTUAL_NETWORK=$(terraform output -raw operations_virtual_network)
+export LOGGING_WORKSPACE=$(terraform output -raw logging_workspace_name)
```

Comment:
```suggestion
export REGISTRY_NAME="$(terraform output -raw operations_container_registry_login_server)"
export TF_VAR_resource_group_name="$(terraform output -raw operations_resource_group_name)"
export TF_VAR_storage_account_name="$(terraform output -raw operations_storage_account_name)"
export SUBNET_ID="$(terraform output -raw operations_deployment_subnet_id)"
export OPERATIONS_VIRTUAL_NETWORK="$(terraform output -raw operations_virtual_network)"
export LOGGING_WORKSPACE="$(terraform output -raw logging_workspace_name)"
```

Any objection to making these assignments a little safer while we're modifying the lines? In general we probably want to get better about quoting things throughout the script